src/main/java/com/efficio/fieldbook/web/trial/controller/ImportGermplasmListController.java

```diff
@@ -210,817 +100,30 @@ public String show(@ModelAttribute("importGermplasmListForm") final ImportGermpl
 	@Transactional
 	public String nextScreen(@ModelAttribute("importGermplasmListForm") final ImportGermplasmListForm form,
 			final BindingResult result, final Model model, final HttpServletRequest req) {
-		// start: section for taking note of the check germplasm
-		boolean isDeleteObservations = false;
-
-		// if we have no germplasm list available for the study, skip this
-		// validation flow
-		if (null != this.userSelection.getImportedGermplasmMainInfo()
-				&& null != this.userSelection.getImportedGermplasmMainInfo().getImportedGermplasmList()) {
-			this.assignPlotNumber(form);
-			isDeleteObservations = true;
-		}
-
 		this.userSelection.setMeasurementRowList(null);
 		this.userSelection.getWorkbook().setOriginalObservations(null);
 		this.userSelection.getWorkbook().setObservations(new ArrayList<>());
+
+		for(MeasurementVariable mv: userSelection.getWorkbook().getFactors()) {
+			//Set Factor's Operation to null since it's autosaved
+			mv.setOperation(null);
+		}
 		this.fieldbookService.createIdCodeNameVariablePairs(this.userSelection.getWorkbook(),
 				AppConstants.ID_CODE_NAME_COMBINATION_STUDY.getString());
 		this.fieldbookService.createIdNameVariablePairs(this.userSelection.getWorkbook(),
 				new ArrayList<SettingDetail>(), AppConstants.ID_NAME_COMBINATION.getString(), true);
 		final int studyId = this.dataImportService.saveDataset(this.userSelection.getWorkbook(), true,
-				isDeleteObservations, this.getCurrentProject().getUniqueID(),
```

Comment:
Logged this tech backlog [ticket](https://ibplatform.atlassian.net/browse/IBP-4129) to cleanup data import service given we auto-save (API) a lot of data now